react_native/README.md

```diff
@@ -16,10 +16,12 @@ Follow the environment setup instructions found in the official React Native doc
 
 #### Install dependencies
 
-Ensure your Node version is >= 16.14.0. Next, run `npm install` in the **TinyQuickstartReactNative/** folder.
+Ensure you're using Node 20; you can run `nvm use` to make sure you are using a compatible version of Node for this project. Next, run `npm install` in the **TinyQuickstartReactNative/** folder.
 
 Navigate to the **ios/** folder and run `pod install` to install all necessary iOS dependencies.
 
+If `pod install` fails, you may need to first install the required version of Ruby and/or cocoapods. You can use `rbenv` to manage your Ruby version and then `rbenv install 3.3.0`. To install cocoapods, run `sudo gem install cocoapods`.
```

Comment:
Since there is now a gemfile. You could also just instruct users to run `bundle install` prior to pod install. This will install the correct version of cocoapods and active support.

Reply:
:zap: